sql/item_mods.sql

```diff
@@ -52119,6 +52119,18 @@ INSERT INTO `item_mods` VALUES (23795,1,1); -- DEF: 1
 -- Ziamet Nails
 INSERT INTO `item_mods` VALUES (23796,1,1); -- DEF: 1
 
+-- Poroggo Cass. +1
+INSERT INTO `item_mods` VALUES (23804,15,20);   -- Fire Resistance
```

Comment:
Please update the Resistance ` -- FIRE_RES: 20 ` and to others so meets the format

Reply:
All set 👍

Reply:
Resist mods were renamed. FIRE_MEVA